tailwind.config.ts

```diff
@@ -17,15 +15,21 @@ const config = {
           colors: {
             primary: {
               DEFAULT: '#f97316',
-              foreground: '#000000',
+              foreground: '#0a0a0a',
+            },
+            background: {
+              DEFAULT: '#f9f9f9',
             },
           },
         },
         dark: {
           colors: {
             primary: {
               DEFAULT: '#ea580c',
-              foreground: '#ffffff',
+              foreground: '#f9f9f9',
+            },
+            background: {
+              DEFAULT: '#0a0a0a',
```

Comment:
is there a reason for these? The integrity of NextUI's colors change with these. The front page's nice white color changes to a weird gray.

Compare this PR's frontpage: https://blog-v3-3hefb7zmm-kir-dev.vercel.app/

And the old homepage: https://kir-dev.hu/